clang-tools-extra/clang-tidy/cppcoreguidelines/PreferMemberInitializerCheck.cpp

```diff
@@ -118,45 +118,45 @@ static void updateAssignmentLevel(
   }
 }
 
-static std::pair<const FieldDecl *, const Expr *>
+static std::optional<std::pair<const FieldDecl *, const Expr *>>
```

Comment:
Consider changing this std::pair into normal struct, in such way there will be no need to make aliases to fields later in check method